app/src/main/java/com/github/h3lp3rs/h3lp/database/MockDatabase.kt

```diff
@@ -182,28 +179,29 @@ class MockDatabase : Database {
 
     @SuppressLint("RestrictedApi")
     override fun <T> addEventListener(
-        key : String?,
-        type : Class<T>,
-        onChildAdded : ((T) -> Unit)?,
-        onChildRemoved : (String) -> Unit
+        key: String?,
+        type: Class<T>,
+        onChildAdded: ((T) -> Unit)?,
+        onChildRemoved: (String) -> Unit

```

Comment:
The onChildRemoved callback is never used which contradicts the Database API, it could be added to a map of onChildRemoved listeners which are just checked and triggered in the delete method

Reply:
Yes but it is not used in our SimpleDBForum yet, so I had no way to know how it works on Firebase :eyes: 
Thinking about coverage :brain: